### ISSUE TITLE
After repaching not able to install app

### ISSUE THREAD

**Issue by reikano** [ryanking13/android-SSL-unpinning#3]
After repaching not able to install app. App name: veve

**Comment by ryanking13** [comment]
@reikano Yes, I was able to reproduce your problem with `com.ecomi.veve`.
Just unpacking and repacking it with doing nothing else makes the application uninstallable.
I'm not sure why it happens for now. I think it is needed to check the behavior of `apktool`.

**Comment by huuhuybn** [comment]
it not working with xapk file . pls help
